src/libgetenv/src/getenv.cpp

```diff
@@ -322,7 +326,11 @@ void applyOptions()
 	if ((k = ksLookupByName(elektraConfig, "/env/option/clearenv", 0)))
 	{
 		LOG << "clearing the environment" << endl;
+#ifdef HAVE_CLEARENV
 		clearenv();
+#else
+		environ = NULL;
```

Comment:
Mihael Pranjić wrote:
> I trusted the man page
> [here](http://man7.org/linux/man-pages/man3/clearenv.3.html) it states that
> `environ = NULL;` should do. I'll look into this if everything else works

I also read that and think the NOTES are really strange (if the statements 
about "security" and "might suffice" are independent) or wrong.

Fixing it is not urgent, because I think libgetenv won't work on Mac OS X 
anyway.